packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -575,11 +575,11 @@ export class CdkToolkit {
     const startSynthTime = new Date().getTime();
     const stackCollection = await this.selectStacksForDeploy(options.selector, true);
     const elapsedSynthTime = new Date().getTime() - startSynthTime;
-    print('\n✨  Synthesis time: %ss\n', formatTime(elapsedSynthTime));
+    print(`\n✨  Synthesis time: ${formatTime(elapsedSynthTime)}s\n`);
 
     if (stackCollection.stackCount === 0) {
       // eslint-disable-next-line no-console
-      console.error('No stacks selected');
+      error('No stacks selected');
```

Comment:
remove eslint comment above. please find all the other instances, i'm not going to comment on them all.